mmv1/third_party/terraform/services/apphub/data_source_apphub_discovered_workloads.go

```diff
@@ -0,0 +1,177 @@
+                                                        Type:     schema.TypeList,
+                                                        Computed: true,
+                                                        Elem: &schema.Resource{
+                                                            Schema: map[string]*schema.Schema{
+                                                                "uri":{
+                                                                    Type: schema.TypeString,
+                                                                    Computed: true,
+                                                                },
+                                                            },
+                                                        },
+                                                },
+                                                "workload_properties": {
+                                                        Type:     schema.TypeList,
+                                                        Computed: true,
+                                                        Elem: &schema.Resource{
+                                                            Schema: map[string]*schema.Schema{
+                                                                "gcp_project":{
+                                                                    Type: schema.TypeString,
+                                                                    Computed: true,
+                                                                },
+                                                                "location":{
+                                                                    Type: schema.TypeString,
+                                                                    Computed: true,
+                                                                },
+                                                                "zone":{
+                                                                    Type: schema.TypeString,
+                                                                    Computed: true,
+                                                                },
+                                                            },
+                                                        },        
+                                                },
+                                        },
+                                },
+                        },
+        },
+    }
+}
+
+func dataSourceApphubDiscoveredWorkloadRead(d *schema.ResourceData, meta interface{}) error {
+        config := meta.(*transport_tpg.Config)
+        userAgent, err := tpgresource.GenerateUserAgentString(d, config.UserAgent)
+        if err != nil {
+                return err
+        }
+
+        url, err := tpgresource.ReplaceVars(d, config, "{{ApphubBasePath}}projects/{{project}}/locations/{{location}}/discoveredWorkloads:lookup?uri={{workload_uri}}" )
```

Comment:
I don't find the GA API doc. 

Is the is the GA GET endpoint? It is different with the alpha GET endpoint.
https://cloud.google.com/app-hub/docs/reference/rest/v1alpha/projects.locations.discoveredServices/get

Reply:
Shared it offline.